PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -11,42 +11,49 @@
 )
 
 patLowPtElectrons = patElectrons.clone(
-    # input collections
+
+    # Input collection
     electronSource = sourceElectrons,
-    genParticleMatch = cms.InputTag("lowPtElectronMatch"),
-    # overrides
-    addElectronID = cms.bool(False),
-    addGenMatch = cms.bool(True),
-    addMVAVariables = cms.bool(False),
-    addPFClusterIso = cms.bool(False),
-    electronIDSources = cms.PSet(),
-    computeMiniIso = cms.bool(False),
-    isoDeposits = cms.PSet(),
-    isolationValues = cms.PSet(),
-    isolationValuesNoPFId = cms.PSet(),
-    miniIsoParamsB = cms.vdouble(),
-    miniIsoParamsE = cms.vdouble(),
-    usePfCandidateMultiMap = cms.bool(False),
-    # embedding
-    embedBasicClusters          = cms.bool(False),
-    embedGenMatch               = cms.bool(False),
-    embedGsfElectronCore        = cms.bool(False),
-    embedGsfTrack               = cms.bool(False),
-    embedHighLevelSelection     = cms.bool(False),
-    embedPFCandidate            = cms.bool(False),
-    embedPflowBasicClusters     = cms.bool(False),
-    embedPflowPreshowerClusters = cms.bool(False),
-    embedPflowSuperCluster      = cms.bool(False),
-    embedPreshowerClusters      = cms.bool(False),
-    embedRecHits                = cms.bool(False),
-    embedSeedCluster            = cms.bool(False),
-    embedSuperCluster           = cms.bool(False),
-    embedTrack                  = cms.bool(True),
-    )
+
+    # MC matching
+    genParticleMatch = "lowPtElectronMatch",
+
+    # Electron ID
+    addElectronID = True,
+    electronIDSources = dict(
+        unbiased = cms.InputTag("rekeyLowPtGsfElectronSeedValueMaps:unbiased"),
+        ptbiased = cms.InputTag("rekeyLowPtGsfElectronSeedValueMaps:ptbiased"),
+        ID       = cms.InputTag("lowPtGsfElectronID"),
+    ),
```

Comment:
Some comments from me below ...

From a RECO perspective, this PR won't change the bParking content.

From a miniAOD perspective, this PR won't change the bParking content, but currently it will change how this content is accessed. 

Specifically, in the current HEAD of 10_6_X, the slimmedLowPtElectrons are stored along with several associated RECO collections (lowPtGsfElectronCores, lowPtGsfTracks, etc...). This PR will leave the content of those collections unchanged, but it will deprecate these collections for miniAOD and instead embed their content directly within the pat::Electron.

If there is a reprocessing of the BParking data sets in 10_6_X, then it is my opinion that the bParking era should continue to be used.

And here are some further comparative comments on the anticipated changes to 10_6_X due to this PR.

In the current HEAD of CMSSW_10_6_X, the bParking modifier is used in:
- [lowPtGsfElectronSeeds_cfi](https://github.com/cms-sw/cmssw/blob/CMSSW_10_6_X/RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSeeds_cfi.py#L50-L53) to lower the pT threshold to 0.5 GeV and loosen the BDT threshold to "Very Loose",
- [MicroEventContent_cff](https://github.com/cms-sw/cmssw/pull/32372/files) to add the slimmedLowPtElectrons and associated RECO collections to miniAOD.

In this PR:
- for [lowPtGsfElectronSeeds_cfi](https://github.com/bainbrid/cmssw/blob/LowPtElectrons_userFloats_106X/RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSeeds_cfi.py#L50-L53), the same bParking modifier logic will be applied (0.5 GeV and "Very Loose"),
- for [MicroEventContent_cff](https://github.com/bainbrid/cmssw/blob/LowPtElectrons_userFloats_106X/PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py#L92-L94), the slimmedLowPtElectrons (and the associated gsfTracksOpenConversions RECO collection) will  now be saved to miniAOD _by default_. Also, as stated above, the information from the deprecated RECO collections will be embedded in the low-pT pat::Electron.
- in [lowPtElectronSelector_cfi.py](https://github.com/bainbrid/cmssw/blob/LowPtElectrons_userFloats_106X/PhysicsTools/PatAlgos/python/selectionLayer1/lowPtElectronSelector_cfi.py#L12-L14), the bParking modifier is used to ensure no selection is applied (to be consistent with the bParking miniAOD content in 10_2_X)

Hopefully this is helpful ...

Reply:
> @silviodonato @rappoccio please clarify/confirm if we need to consider possible `bParking` production in the 10_6_X context

There are currently no official requests for UL of the bParking. That being said, it is still an option that may happen in early 2021.

Reply:
> From a miniAOD perspective, this PR won't change the bParking content, but currently it will change how this content is accessed.

apparently it is not the case: this PR is adding electronID variables to the lowPtElectrons running with the `bParking` modifier, as can be seen in the outputs of wf 1304.181 and 136.898
This change is triggered by the lines highlighted in this comment subthread (`addElectronID = True ` ... )

Since only new variables are saved in the products and apparently these variables were already computed for other reasons, I think that it should be acceptable to keep this change.